src/o365/graph/commands/schemaextension/schemaextension-set.ts

```diff
@@ -0,0 +1,247 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  owner: string;
+  description?: string;
+  status?: string;
+  targetTypes?: string;
+  properties?: string;
+}
+
+class GraphSchemaExtensionSet extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_SET}`;
+  }
+
+  public get description(): string {
+    return 'Updates a Microsoft Graph schema extension';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      cmd.log(`Updating schema extension with id '${args.options.id}'...`);
+    }
+
+    // The default request body always contain owner
+    const body = {
```

Comment:
To avoid typos and simplify future refactorings, we should define a type for body like:

```ts
    const body: {
      owner: string;
      description?: string;
      status?: string;
      targetTypes?: string[];
      properties?: any;
    } = {
      owner: args.options.owner
    };
```